### ISSUE TITLE
Update decision logic for BotTests to use structured machine learning

### ISSUE THREAD

**Issue by joekir** [safety-net/galvatron#4]
Build on growing training data to create a function for stochastic gradient descent predictions.
This baseline could be updated from testing feedback or even endpoint callbacks.
